docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -617,160 +743,164 @@ main(){
+    kubectl_wrapper wait deployments "ziti-console" \
         --namespace "${ZITI_NAMESPACE}" \
         --for condition=Available=True \
         --timeout 240s >&3
 
     logDebug "setting default namespace to '${ZITI_NAMESPACE}' in kubeconfig context '${MINIKUBE_PROFILE}'"
-    minikube kubectl --profile "${MINIKUBE_PROFILE}" -- \
-        config set-context "${MINIKUBE_PROFILE}" \
+        kubectl_wrapper config set-context "${MINIKUBE_PROFILE}" \
             --namespace "${ZITI_NAMESPACE}" >&3
 
     #
     ## Ensure OpenZiti Identities and Services are Created
     #
 
-    if  ! ziti edge list identities "name=\"${MINIKUBE_PROFILE}-client\"" --csv \
-        | grep -q "${MINIKUBE_PROFILE}-client"; then
-        logDebug "creating identity ${MINIKUBE_PROFILE}-client"
-        ziti edge create identity device "${MINIKUBE_PROFILE}-client" \
-            --jwt-output-file "/tmp/${MINIKUBE_PROFILE}-client.jwt" --role-attributes httpbin-clients >&3
+    CLIENT_NAME="${MINIKUBE_PROFILE}-client"
+    CLIENT_OTT="$IDENTITIES_DIR/$CLIENT_NAME.jwt"
+
+    if  ! ziti_wrapper edge list identities "name=\"$CLIENT_NAME\"" --csv \
+        | grep -q "$CLIENT_NAME"; then
+        logDebug "creating identity $CLIENT_NAME"
+        ziti_wrapper edge create identity device "$CLIENT_NAME" \
+            --jwt-output-file "$CLIENT_OTT" --role-attributes httpbin-clients >&3
     else
-        logDebug "ignoring identity ${MINIKUBE_PROFILE}-client"
+        logDebug "ignoring identity $CLIENT_NAME"
     fi
 
-    if  ! ziti edge list identities 'name="httpbin-host"' --csv \
-        | grep -q "httpbin-host"; then
-        logDebug "creating identity httpbin-host"
-        ziti edge create identity device "httpbin-host" \
-            --jwt-output-file /tmp/httpbin-host.jwt --role-attributes httpbin-hosts >&3
+    HTTPBIN_NAME="httpbin-host"
+    HTTPBIN_OTT="$IDENTITIES_DIR/$HTTPBIN_NAME.jwt"
+    HTTPBIN_JSON="$IDENTITIES_DIR/$HTTPBIN_NAME.json"
+
+    if  ! ziti_wrapper edge list identities "name=\"$HTTPBIN_NAME\"" --csv \
+        | grep -q "$HTTPBIN_NAME"; then
+        logDebug "creating identity $HTTPBIN_NAME"
+        ziti_wrapper edge create identity device "$HTTPBIN_NAME" \
+            --jwt-output-file "$HTTPBIN_OTT" --role-attributes httpbin-hosts >&3
     else
-        logDebug "ignoring identity httpbin-host"
+        logDebug "ignoring identity $HTTPBIN_NAME"
     fi
 
-    if  ! ziti edge list configs 'name="httpbin-intercept-config"' --csv \
+    if  ! ziti_wrapper edge list configs 'name="httpbin-intercept-config"' --csv \
         | grep -q "httpbin-intercept-config"; then
         logDebug "creating config httpbin-intercept-config"
-        ziti edge create config "httpbin-intercept-config" intercept.v1 \
+        ziti_wrapper edge create config "httpbin-intercept-config" intercept.v1 \
             '{"protocols":["tcp"],"addresses":["httpbin.'"${MINIZITI_INTERCEPT_ZONE}"'"], "portRanges":[{"low":80, "high":80}]}' >&3
```

Comment:
This results in colliding intercepts when miniziti profiles are running concurrently. Consider building the intercept zone with the profile name.

Reply:
This should be resolved.

Reply:
This should be resolved.